cluster/images/conformance/go-runner/tar_test.go

```diff
@@ -73,6 +73,10 @@ func TestTar(t *testing.T) {
 			}
 
 			data, err := readAllTar(tc.outpath)
+			if err != nil {
+				t.Fatalf("Failed to read tarball: %v, err")
```

Comment:
^^ misquoted, should be
`t.Fatalf("Failed to read tarball: %v", err)`

Reply:
I fixed it.

Reply:
Thank you for your suggestion.